### DISCUSSION TITLE
Preline Pro

### DISCUSSION THREAD

**Discussion by jjsmall009** [htmlstreamofficial/preline#43] [General]
I recently came across this library and am pretty blown away by how awesome it is.

I notice there's a pro version that's coming soon. Are there any details on what it will entail? Will the current components and functionality be changing or get locked behind the pro version?

**Comment by Mr-Zafar** [comment]
Thanks for checking out Preline UI!

At the moment, we can not reveal anything about the Pro version. However, the current free version will always remain free and open source. 

Subscribe to [Preline.co](https://preline.co/) to receive updates when the Pro version is launched.
